### DISCUSSION TITLE
Contribute translations

### DISCUSSION THREAD

**Discussion by jannesblobel** [jorge-menjivar/unSAGED#94] [Q&A] [answered]
Hey guys, 
I would love to contribute German as a language, and I also saw that you are using next-i18next. But I need help finding your locals or i18n folder. :(

**Comment by Bortus-AI** [comment, marked as answer]
Are you looking for this folder?
packages/unsaged/public/locales

**Reply by jannesblobel** [reply]
Wow, so so sorry, it was hard to find.

**Reply by Bortus-AI** [reply]
Don't sweat it. I had to use the search tool 😂
